### PR TITLE
Add Sign Message Button for Nano and Hub

### LINKED ISSUE #12: Add button for signing message
Based on this https://github.com/LiskHQ/lisk-hub/issues/457#issuecomment-381079550

### PR DESCRIPTION
**Pull request by hirishh**
Nano and Hub share different url for sign-message.
Moreover, lisk-nano doesn't have a message-prefill option, so it will just open sign message page.
closes #12

### CODE REVIEW

**Review by alepop — APPROVED** [review]
LGTM 👍

File: src/index.html [hunk 1]
```diff
@@ -25,6 +25,9 @@
         <lisk-button-send amount=100 recipient="15015136092749848942L" button-title="Send Me LSK for food"></lisk-button-send>
         <lisk-button-vote votes="goodPerson1,goodPerson2,goodPerson3" unvotes="badPerson1,badPerson2,badPerson3"></lisk-button-vote>
         <lisk-button-vote votes="goodPerson1,goodPerson2,goodPerson3" button-title="Vote for ME"></lisk-button-vote>
+        <lisk-button-sign type="hub" message="sign with hub" button-title="Sign by using Lisk Hub!"></lisk-button-sign>
+        <lisk-button-sign message="sign with hub 2"></lisk-button-sign>
```
Comment (nerdvibe) [inline src/index.html:29]: What is the 2?

**Comment by nerdvibe** [comment]
Will review soon :)

File: src/components/utils/index.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import checURI from '@alepop/check-for-uri-handler';
```
Comment (Isabello) [inline src/components/utils/index.ts]: not specifically your fault, but this should be `checkURI`, probably.

File: src/index.html (hunk 1 above)
Comment (hirishh) [reply src/index.html:29]: It helps during real test to have a visual feedback that both are working

Comment (nerdvibe) [reply src/index.html:29]: ah lol true, that's the index.html :)

File: src/components/utils/index.ts (hunk 2 above)
Comment (alepop) [reply src/components/utils/index.ts]: ooops

**Comment by nerdvibe** [comment]
Nice nice lgtm 👍

**Comment by alepop** [comment]
@hirishh thanks for your contribution 👍

**Pull request merged**